src/accelerate/utils/modeling.py

```diff
@@ -591,6 +591,10 @@ def get_balanced_memory(
             ]
         )
 
+    if num_devices == 1:
+        # We cannot do low_zero on just one GPU
+        low_zero = False
```

Comment:
Should we add a `logging.info` or `logging.warn` here (normal logger) to alert the user that we're silently not letting something happen they want?